src/nodes/Table/TableCell.js

```diff
@@ -58,6 +59,17 @@ export default TableCell.extend({
 		}
 	},
 
+	renderHTML({ HTMLAttributes }) {
+		const attributes = mergeAttributes(this.options.HTMLAttributes, HTMLAttributes)
```

Comment:
Could you elaborate a bit on why this is necessary? I didn't grasp what the `colspan` and `rowspan` attributes are used for before.

Reply:
The colspan and rowspan will be added automatically by tiptap.

Reply:
@mejo- I think this comment should answer you.
https://github.com/nextcloud/text/issues/2267#issuecomment-2410600211